app/platform/fabric/sync/SyncService.js

```diff
@@ -533,7 +533,7 @@ class SyncServices {
 				);
 				if (txid && txid !== '') {
 					const validation_codes =
-						block.metadata.metadata[block.metadata.metadata.length - 1];
+						block.metadata.metadata[2];
 					const val_code = validation_codes[i];
```

Comment:
I suggest using predefined constant `fabprotos.common.TRANSACTIONS_FILTER` rather than instant value. @wuhua666 
https://github.com/hyperledger/fabric-sdk-node/blob/8156c0d58a9752c8931212b4d57c15fe65595ac4/fabric-protos/protos/common/common.proto#L45